Rakefile

```diff
@@ -242,7 +247,7 @@ Rake::Task['arm:regen_individual_profiles'].enhance do
   Rake::Task['arm:regen_individual_require_files'].invoke
 end
 
-Rake::Task['arm:regen'].enhance do
+Rake::Task['arm:regen_sdk_versions'].enhance do
```

Comment:
change is ok. I think I made a comment on this before, on what's the benefit of enhance? could we be consistent and have the regen_sdk_versions task just call regen_individual_require_files as a pre-req?

Reply:
Actually, You cannot make regen_individual_require_files a pre requisite. If you execute regen_individual_require_files before regen_sdk_versions task, the require file will be empty. Because, in order to generate the require files, the sdk versions must be generated. i.e. you need to execute regen_individual_require_files task AFTER the regen_sdk_versions task. Now, it is possible to create a third task and make both of them as pre reqs. But, according to rake specification the order is not guaranteed. So, enhancement is the only option. Also, enhancement is standard design option provided by Rake and I believe we must start using it.